tests/end2end/playwright/globals.js

```diff
@@ -12,6 +14,34 @@ import { URLSearchParams } from 'url';
  * @typedef {number} int
  */
 
+/**
+ * The file path
+ * @var string
+ */
+const __filename = fileURLToPath(import.meta.url);
+
+/**
+ * The file directory path
+ * @var string
+ * @see https://nodejs.org/docs/latest-v15.x/api/esm.html#esm_no_filename_or_dirname
+ * @see https://stackoverflow.com/questions/8817423/why-is-dirname-not-defined-in-node-repl
+ * @example
+ * import { fileURLToPath } from 'url';
+ * import { dirname } from 'path';
+ * const __filename = fileURLToPath(import.meta.url);
+ * const __dirname = dirname(__filename);
+ */
+export const __dirname = path.dirname(__filename);
+
+/**
+ * Get the auth storage state path
+ * @param {string} name The file name without extension
+ * @returns {string} The path to auth storage state path
+ */
+export function getAuthStorageStatePath(name) {
+    return path.join(__dirname, './.auth/' + name + '.json')
```

Comment:
Side question, should we prefer string templates ? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#syntax

@nboisteault ?

Reply:
I prefer string templates when there are multiple line breaks. In this case I don't have any preference.

Reply:
Thanks @Gustry to pointed it. The right syntax is not string template but more than 2 parameters for `path.join` like `path.join(__dirname, '.auth', name + '.json')`